fr.ts

```diff
@@ -4118,7 +4118,7 @@ Veuillez visiter http://lmms.sf.net/wiki pour la documentation de LMMS.</transla
     <message>
         <source>The JACK server seems to be shuted down.</source>
         <extracomment>When JACK(JACK Audio Connection Kit) disconnects, it will show the following message (dialog message)</extracomment>
-        <translation type="unfinished"/>
+        <translation>Quand JACK(JACK Audio Connection Kit) se déconnecte, il affichera le message suivant (message de dialogue)</translation>
```

Comment:
Sorry, but I think you were translating the comments which we clarify the translation strings.
For instance, in this case:

```XML
<extracomment>When JACK(JACK Audio Connection Kit) disconnects, it will show the following message (dialog message)</extracomment>
```

is a comment, which is for you to understand, and

```XML
<source>The JACK server seems to be shuted down.</source>
```

This is a source string that you need to translate.